README.md

```diff
@@ -69,7 +69,7 @@ Type `?heppcat` for the [docstring](https://docs.julialang.org/en/v1/manual/docu
 help?> heppcat
 search: heppcat HePPCAT HePPCATModel
 
-  heppcat(Y,k,iters;init=homppca(Y,k))
+  heppcat(Y,k,iters;init=homppca(Y,k), vknown=false, eps=0)
```

Comment:
maybe we don't need to show all the kwargs in the readme?

Reply:
Right now the README is just showing the docstring, so it should probably show all the options.

That said, I don't think we have to use the docstring there - doing so was just a convenient way of showing the functionality and showing users how to get help. On a related note, at some point, I'd really like to add docs with some examples.